.eslintrc.js

```diff
@@ -78,7 +78,7 @@ module.exports = {
       },
     },
     {
-      files: ['**/tests/**/*.ts', '**/tests/**/*.tsx', '**/loom.config.ts'],
+      files: ['**/tests/**/*.ts', '**/tests/**/*.tsx', '**/loom.config.ts', '**/*.test-d.ts', '**/*.test.d.tsx'],
```

Comment:
Should that  `**/*.test.d.tsx` be  `**/*.test-d.tsx`?

```suggestion
      files: ['**/tests/**/*.ts', '**/tests/**/*.tsx', '**/loom.config.ts', '**/*.test-d.ts', '**/*.test-d.tsx'],
```

Though do we even need '**/*.test-d.tsx'? When would we want these type-checking files to contain JSX content?